neuror/sanitize.py

```diff
@@ -56,16 +78,13 @@ def sanitize_all(input_folder, output_folder):
     '''
     set_maximum_warnings(0)
 
-    errored_paths = list()
-    for path in tqdm(list(iter_morphologies(Path(input_folder)))):
-        relative_path = path.relative_to(input_folder)
-        output_dir = output_folder / relative_path.parent
-        if not output_dir.exists():
-            output_dir.mkdir(parents=True)
-        try:
-            sanitize(path, output_dir / path.name)
-        except (MorphioError, CorruptedMorphology):
-            errored_paths.append(str(path))
+    morphologies = list(iter_morphologies(Path(input_folder)))
+    args = ((morphology, input_folder, output_folder) for morphology in morphologies)
```

Comment:
here you can give the iter_morphologies to pool.imap directly

Reply:
Yes, as you said in the next comment, I create a list to know the number of items for `tqdm`